### PR TITLE
Update graph filter readme

### LINKED ISSUE #156: Update README
The README is outdated. Record the new layout screens and add the new functionalities.

### PR DESCRIPTION
**Pull request by rosaigor138**
Closes #156

### CODE REVIEW

File: resources/gifs/project_new_graph.gif
Comment (atilaajones) [inline resources/gifs/project_new_graph.gif:1]: Is there a problem with this gif? I can't see it.

File: README.md [hunk 1]
```diff
@@ -121,20 +121,47 @@ Here you can load the graphs for filtering. The file format of the entry is .g6
...
-- PDF
-- PNG
-- Latex (TikZ)
-- graph6 (.g6 or .txt)
-- xlsx
+  > To navigate between the graphs returned from the filtering or opened from a .g6, .txt file, you can use the combo box or navigation arrows at the top of the window.
 
+#### Tool Bar
+
+The toolbar gives you access to some of the software's features, such as applying certain [operations](https://github.com/GraphFilter/GraphFilter/wiki/Features#operations), exporting, inserting a [universal vertex](https://github.com/GraphFilter/GraphFilter/wiki/Features#operations), saving the edited graph or deleting it. These functions refer to the graph on the screen. You can also create a new commonly used graph to work with the software. In the [dictionary tab](https://github.com/GraphFilter/GraphFilter/wiki/Dictionary#graphs), you can see all the new graphs options with their definitions and parameters.
```
Comment (atilaajones) [inline README.md:134]: I suggest change "commonly" to "well-known"

**Review by atilaajones — CHANGES_REQUESTED** [review]

File: resources/gifs/project_new_graph.gif
Comment (fsoupimenta) [reply resources/gifs/project_new_graph.gif:1]: Are you trying to see it on Github? I tried to see it from my machine and from here, and both showed up normally.

File: README.md [hunk 2]
```diff
@@ -30,8 +30,7 @@ providing for visualization a filtered list of graphs according to the propertie
...
 
-The software has the purpose of filtering a list of graphs through conditions customized by the user, without requiring programming knowledge. After the process, the filtered graphs are returned and the user can analyze them according to his cientific research. This user will have access to plot of the graphs and the calculation of several invariants, at the user's choice.
-
+In version 2.0 and before the software has the purpose of filtering a list of graphs through conditions customized by the user, without requiring programming knowledge. After the process, the filtered graphs are returned and the user can analyze them according to his cientific research. This user will have access to plot of the graphs and the calculation of several invariants, at the user's choice. Now in version 3.0, the software has received updates to improve usability, such as a file tree for handling graph files, on-screen graph drawing functions, saving and deleting, and the creation of empty projects as well as the creation of graphs with known definitions.
```
Comment (atilaajones) [inline README.md:33]: I think it's unnecessary to write about the difference in versions here (2.0 and 3.0), You can talk directly about the two main functionalities.

File: README.md [hunk 3]
```diff
@@ -40,6 +39,9 @@ The software has the purpose of filtering a list of graphs through conditions cu
 + Intuitive layout, does not require programming knowledge;
 + Support for several graph invariants;
 + It helps the researcher in Graphs to test conjectures and results.
++ Visualize, draw and manipulate Graphs
++ Create well-known Graph layouts
++ Manipulate files with an file tree
```
Comment (atilaajones) [inline README.md:44]: Maybe: _Manipulate files with an tree file manager_?

**Review by atilaajones — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (rosaigor138) [reply README.md]: I've restructured the text as you requested, I'd like you to take a look at the new text

File: README.md [hunk 4]
```diff
@@ -30,7 +30,7 @@ providing for visualization a filtered list of graphs according to the propertie
...
 
-In version 2.0 and before the software has the purpose of filtering a list of graphs through conditions customized by the user, without requiring programming knowledge. After the process, the filtered graphs are returned and the user can analyze them according to his cientific research. This user will have access to plot of the graphs and the calculation of several invariants, at the user's choice. Now in version 3.0, the software has received updates to improve usability, such as a file tree for handling graph files, on-screen graph drawing functions, saving and deleting, and the creation of empty projects as well as the creation of graphs with known definitions.
+The software has the function of filtering a list of graphs using conditions customized by the user, without requiring programming knowledge. It also has the function of manipulating, drawing and creating graphs following known patterns. After filtering, the software will open the list of filtered graphs to be analyzed according to your scientific research and have invariants applied to them, as well as being able to manipulate these graphs or create new ones.
```
[the remark this answers is not in the document]
Comment (atilaajones) [reply README.md:33]: I suggest to use the text available on [page](http://sistemas.jf.ifsudestemg.edu.br/graphfilter/). What do you think? See below.

The Graph Filter is a software to help graph researchers, with a simple and intuitive interface. It allows you to draw, save and manipulate graphs with the mouse and keyboard. It provides more than 170 invariants (spectral and structural) and some graph operations, which are calculated in a few clicks.

Another purpose of the software is to filter graphs: the user enters conditions and equations, attaches a list of up to thousands of graphs, then the program filters the graphs that meet the conditions. An important tool for drawing up results and finding counterexamples. You can also export your graphs in the following formats: gml, graph6, png, pdf or latex.

The program is supported by any Windows, Linux and Mac. No installation is required,[ just download and run](http://sistemas.jf.ifsudestemg.edu.br/graphfilter/download).

**Review by atilaajones — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (rosaigor138) [reply README.md]: Okay, I agree and changed as you requested

**Review by atilaajones — CHANGES_REQUESTED** [review]
I have two suggestions:

- [x] Change the topic _"Release"_ to _"Run from executable (release)"_.  And _"Source"_ code to _"Run from source code"_

- [x] Update the list of export files to:
Graph (.gml)
Graph6 (.g6 or .txt)
Image (.png or .pdf)
Latex Tikz (.tex)
Sheets with ghrap (.xlsx)

**Review by atilaajones — APPROVED** [review]

**Pull request merged**